README.md

```diff
@@ -1,40 +1,16 @@
 <a name="readme-top"></a>
 
-<!--
-HOW TO USE:
-This is an example of how you may give instructions on setting up your project locally.
-
-Modify this file to match your project and remove sections that don't apply.
-
-REQUIRED SECTIONS:
-- Table of Contents
-- About the Project
-  - Built With
-  - Live Demo
-- Getting Started
-- Authors
-- Future Features
-- Contributing
-- Show your support
-- Acknowledgements
-- License
-
-OPTIONAL SECTIONS:
-- FAQ
-
-After you're finished please remove all the comments and instructions!
--->
+
 
 <div align="center">
-  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
-  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+
+  <img src="./images/school-logo-small.png" alt="logo" width="140"  height="auto" />
```

Comment:
[OPTIONAL] 
- The logo on your README.md is not showing up as seen in the image below.
![image](https://user-images.githubusercontent.com/45460192/219823626-d27241a2-578b-4b14-8542-4d8f700d4dc1.png)